packages/base/card-api.gts

```diff
@@ -1783,6 +1786,188 @@ class DefaultCardDefTemplate extends GlimmerComponent<{
   </template>
 }
 
+class DefaultEmbeddedTemplate extends GlimmerComponent<{
+  Args: {
+    cardOrField: typeof BaseDef;
+    model: CardDef;
+    fields: Record<string, new () => GlimmerComponent>;
+    context?: CardContext;
+  };
+}> {
+  <template>
+    <div class='embedded-template'>
+      <div class='container'>
+        <div class='contents'>
+          {{#if @model}}
+            <div class='thumbnail-section'>
+              <div
+                class='card-thumbnail'
+                style={{cssUrl 'background-image' @model.thumbnailURL}}
+              >
+                {{#unless @model.thumbnailURL}}
+                  <div
+                    class='card-thumbnail-text'
+                    data-test-card-thumbnail-text
+                  >{{cardTypeDisplayName @model}}</div>
+                {{/unless}}
+              </div>
+            </div>
+            <div class='thumbnail-subsection'>
+              <div class='thumbnail-subsection'>
+                <h3
+                  class='card-title'
+                  data-test-card-title
+                >{{@model.title}}</h3>
+              </div>
+              <div class='thumbnail-subsection'>
+                <h4
+                  class='card-display-name'
+                  data-test-card-display-name
+                >{{cardTypeDisplayName @model}}</h4>
+              </div>
+            </div>
+          {{else}}
+            {{! empty links-to field }}
+            <div data-test-empty-field class='empty-field'></div>
```

Comment:
probably we should show something reasonable here. it seems this was always rendering for empty `<@field.emptyLinkValue/>`.